sdk/cosmos/azure-cosmos-spark_3_2-12/Samples/Scala/filteringDuringMigration.scala

```diff
@@ -0,0 +1,159 @@
+// Databricks notebook source
+val cosmosEndpoint = "<cosmos-endpoint>"
+val cosmosMasterKey = "<cosmos-master-key>"
+val cosmosDatabaseName = "ContosoHospital"
+val cosmosContainerName = "Patient"
+
+
+// Patient Document -- partition key: patientId
+// {
+//   "id": "9c9a1156-e936-40f3-a442-e9528b55a2fb",
+//   "patientId": "423ab2cf-dd1c-4404-8524-86cee045c179",
+//   "patientName": "John Doe",
+//   "doctorId" : "629f49da-9cfc-45a4-8e1c-d4f8b7ab1f4e",
+//   "doctorName": "Sung Ondricka"
+// }
+
+
+//-----filtering examples-with-schema-inference-disabled---------------------
+
+val TargetContainerName = "CopyContainer"
+val checkpointLocation = "/tmp/streaming_checkpoint"
+
+val changeFeedCfg = Map("spark.cosmos.accountEndpoint" -> cosmosEndpoint,
+  "spark.cosmos.accountKey" -> cosmosMasterKey,
+  "spark.cosmos.database" -> cosmosDatabaseName,
+  "spark.cosmos.container" -> cosmosContainerName,
+  "spark.cosmos.read.inferSchema.enabled" -> "false",   
+  "spark.cosmos.changeFeed.startFrom" -> "Beginning",
+  "spark.cosmos.changeFeed.mode" -> "Incremental",
+  "spark.cosmos.changeFeed.itemCountPerTriggerHint" -> "10000"
```

Comment:
Unless the target throughput is extremely small 10_000 is pretty much always too low. It would mean roughly for every 10_--- documents a new micro batch is created. often will leave only one batch execution on each physical partition. Usually 100K - 500K is a good number when the goal is to get quick feedback over progress. 1-5 million when the to-be-migrated data volume is large

Reply:
changed it to 100000